subsys/bluetooth/shell/audio.c

```diff
@@ -1409,6 +1409,36 @@ static int cmd_term_broadcast_sink(const struct shell *sh, size_t argc,
 }
 #endif /* CONFIG_BT_AUDIO_BROADCAST_SINK */
 
+static int cmd_context(const struct shell *sh, size_t argc, char *argv[])
+{
+	int err = 0;
+	enum bt_audio_dir dir;
+	enum bt_audio_context ctxt;
+
+	if (!strcmp(argv[1], "sink")) {
+		dir = BT_AUDIO_DIR_SINK;
+	} else if (!strcmp(argv[1], "source")) {
+		dir = BT_AUDIO_DIR_SOURCE;
+	} else {
+		shell_error(sh, "Unsupported dir: %s", argv[1]);
+		return -ENOEXEC;
+	}
+
+	ctxt = shell_strtoul(argv[2], 10, &err);
```

Comment:
I think this will also be better to decode as hex than base 10, as it is a bitfield

Reply:
Done